ljps-frontend/pages/skill/createskillbutton.js

```diff
@@ -0,0 +1,157 @@
+import axios from "axios";
+import React from "react";
+import { useState, useRef, useEffect } from "react";
+import { validateLength } from "../../util/validation/index";
+
+export default function CreateSkillButton() {
+  const [skillName, setSkillName] = useState("");
+  const [skillDescription, setSkillDescription] = useState("");
+  const [showError, setShowError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const modal = useRef();
+  const toast = useRef();
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    setErrorMessage("")
```

Comment:
Move this setErrorMessage inside the if statement. You are setting the state twice if there is an error.

Reply:
done!